lc-0242-valid-anagram/src/main/java/solution/Solution.java

```diff
@@ -0,0 +1,27 @@
+package solution;
+
+public class Solution {
+    public boolean isAnagram(String s, String t) {
+
+        int[] charArr = new int['z' - 'a' + 1];
+
+        for (char c : s.toCharArray()) {
+            charArr[c - 'a']++;
+        }
+
+        for (char c : t.toCharArray()) {
+            charArr[c - 'a']--;
+        }
+
+        for (int i : charArr) {
+            if (i != 0)
+                return false;
+        }
+
+        return true;
```

Comment:
This works great. I just wanted to introduce you to [declarative vs imperative](https://stackoverflow.com/a/1784702) programming.

The declarative way would look something like:
```
return !ArrayUtils.contains(charArr, 0);
```

Unfortunately Java has very few useful functions for arrays, and this will require using an awkward Stream API or an external library like Apache Commons.

However, I suggest you figure out how to depend on Apache Commons in Gradle for all solutions. It has many useful functions for strings and collections.

See https://stackoverflow.com/a/6188194 for more information.